### DISCUSSION TITLE
the TTS part doesn't work

### DISCUSSION THREAD

**Discussion by kevindup** [mmpneo/simple-obs-stt#3] [General]
Hello, first of all I would like to congratulate you for the work done.
The voice recognition part (or STT) works very well, I can make it work without any problem.
However I have a problem with the TTS, it doesn't work.
When I enter the key and press 'start' nothing happens.
I used the last executable version, the 1.6.0
![Desktop Screenshot 2022 01 21 - 02 10 57 74](https://user-images.githubusercontent.com/98132505/150656439-a4772dd2-999f-49f3-bb6e-561386d7487b.png)

**Comment by mmpneo** [comment]
I'm so sorry, missed email notification. I'll look into that.
